### ISSUE TITLE
Row level query subscription not listening to deletes.

### ISSUE THREAD

**Issue by VaibhavAcharya** [supabase/supabase-js#122]
# Bug report

## Describe the bug

Realtime subscription of row level query is not able to listen to changes of type `delete`

## To Reproduce

Here is a snippet of my code:
```
supabase.from(`posts:author=eq.${user.id}`)
        .on('*', (payload) => {
          console.log('changes: ', payload)
          fetchNewPosts();
        })
        .subscribe()
```

## Expected behavior

It should be able to listen to all types of changes. But it only works for `INSERT` and `UPDATE`.

## Screenshots

None.

## System information

- OS: Windows
- Browser: chrome
- Version of supabase-js: 1.2.1
- Version of Node.js: v15..0.1

## Additional context

None.

**Comment by VaibhavAcharya** [comment]
I researched more and found out that this issue has been fixed in @supabase/realtime `v0.7.11`.
When is this dependency expected to be updated here?
It has been quite a time since that fix was released.😅

**Comment by kiwicopple** [comment]
Hi @VaibhavAcharya, this is already released, but I imagine you are missing the "full" replication:

https://supabase.io/docs/client/subscribe#listening-to-deletes

you will need to run this to listen to deletes: `alter table "your_table" replica identity full;`

I'll close this now but feel free to reopen if it's a bug on our side
